### ISSUE TITLE
Not Recognising Password or Username

### ISSUE THREAD

**Issue by EntropyAssimilated** [Berry-Pool/nami-wallet#97]
Hey folks, I am having issue making transactions with the Nami Wallet. It says my password is incorrect, although I wrote my password down when setting it so relatively confident that it is correct... Regardless, I tried to reset my password but it does not recognise my email address or username when I try to do so. I am signed into the wallet with that username right now...so not quite sure why it is not accepting it as valid. Any assistance on this would be greatly appreciated.
Thank you.

**Comment by pascallapointe** [comment]
Sorry but i'm not following, Nami doesn't use email address nor username.
There is no notion of recovering a password either, and you are never "signed" into the wallet as there is not user account.
The only thing you can do is uninstall/reinstall & restore from the 24 words seed phrase.

Since Nami doesn't use theses features, if you did try to enter a username or email somewhere along with the password, I would then strongly suggest you to quickly change all accounts passwords.

Please provide more information in case a fake NamiWallet scam is going on.

**Comment by EntropyAssimilated** [comment]
Ah okay, shit will do so right away. I managed to restore my wallet anyway
and it is functioning fine. I changed account passwords too.

The place I tried to reset the password was here:
https://auth.nami.io/reset_password . It does look pretty official...but
all details have been changed anyway and will relocate some funds just in
case.

I think that is all there was to it. So, thanks for the response.
